content/cdi-iii/0005-equacoes-ordem-superior.md

```diff
@@ -151,7 +151,7 @@ As soluções $y_1, \dots, y_n$ podem ser calculadas da seguinte forma.
 
      $$
      \begin{darray}{l}
-     t^k e^{at} \sin (bt) = \frac{1}{2i} t^k e^{at} \left(e^{i bt} - e^{i bt} \right) = \frac{1}{2i} t^k e^{(a+ib)t} + \frac{1}{2i} t^k e^{(a - ib)t}\\
+     t^k e^{at} \sin (bt) = \frac{1}{2i} t^k e^{at} \left(e^{i bt} - e^{-i bt} \right) = \frac{1}{2i} t^k e^{(a+ib)t} + \frac{1}{2i} t^k e^{(a - ib)t}\\
```

Comment:
```suggestion
     t^k e^{at} \sin (bt) = \frac{1}{2i} t^k e^{at} \left(e^{i bt} - e^{-i bt} \right) = \frac{1}{2i} t^k e^{(a+ib)t} - \frac{1}{2i} t^k e^{(a - ib)t}\\
```
I think this is also wrong as well, even though is a `+` on the original source. Can you check I'm not crazy?

Reply:
Yes, you are right! I missed that one! ^^'